docs/experimental/index.md

```diff
@@ -0,0 +1,3 @@
+# Experimental
+
+This section documents use of experimental features that are under active development but not part of the main package yet.
```

Comment:
Would it be better to just have the active learning and any other experimental notebooks in the same place but suppress publishing them to the site, or do we want people to read in-progress work online?